Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:latest
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
```

Comment:
This is fine as long as you're sure you want to copy everything in the path over. It might be worthwhile to specify folders if you want to avoid possibly including some things in your production environment that you don't want (or need) to be there.